src/player/Player.php

```diff
@@ -2505,6 +2509,17 @@ public function getCraftingGrid() : CraftingGrid{
 		return $this->craftingGrid;
 	}
 
+	public function getCreativeInventory() : CreativeInventory{
+		return $this->creativeInventory;
+	}
+
+	public function setCreativeInventory(CreativeInventory $inventory) : void{
+		$this->creativeInventory = $inventory;
+		if($this->isConnected()){
+			$this->getNetworkSession()->getInvManager()?->syncCreative();
```

Comment:
This might not be appropriate. If it's called during the login sequence it could lead to a client crash.